src/components/MegaMenu/MegaMenu.tsx

```diff
@@ -62,9 +64,9 @@ export const MegaMenu = ({ onClose, user, changeLanguage, logout }: MegaMenuProp
             <button onClick={doClose} className="flex items-center ml-5">
               <span
                 aria-hidden
-                className={`text-black sm:text-secondary-gray material-symbols-outlined size-32 flex size-[32px] select-none items-center justify-center self-center rounded-full`}
+                className={`text-black sm:text-secondary-gray flex select-none items-center justify-center self-center rounded-full`}
```

Comment:
Not from this PR, but no longer seems to need of backtick-style. Could be changed to 
```tsx
className="text-black sm:text-secondary-gray flex select-none items-center justify-center self-center rounded-full"
```